roles/logs_client/meta/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+dependencies:
+ - role: static_hostname_lookup
+ - role: iptables
+ - role: yum_repos
+   when: repo_manager == 'none'
+ - role: pulp_client
+   when: repo_manager == 'pulp'
```

Comment:
That is a lot. Note that when you do include all these roles as dependencies then they will be executed each time you run the role. Hence, you cannot make a `single_role_playbook` anymore, because it will always run these 4 dependencies before running `logs_client`. Therefore I suggest not to add these here, but to document that things like the firewall need to be configured properly first and hence that these 4 deps must come first before `logs_client` in `single_group_playbooks`

Reply:
ok, removed and added documentation. Let's just hope that people will be reading it.